README.md

```diff
@@ -256,4 +273,12 @@ grpcurl -plaintext -d '{}'  localhost:7100 eigenlayer.sidecar.api.v1.Rpc/GetBloc
 ### Get the stateroot at a block height
 
 ```bash
+#Example for mainnet ethereum
 grpcurl -plaintext -d '{ "blockNumber": 1140438 }'  localhost:7100 eigenlayer.sidecar.api.v1.Rpc/GetStateRoot
+```
+
+```bash
+# Example for holesky testnet
```

Comment:
This comment is inaccurate since this request is relevant for both holesky and mainnet. Better to just leave it out entirely